src/pages/data-stack/business-intelligence.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import ReaderView from 'components/ReaderView'
+import { customerDataInfrastructureNav } from '../../hooks/useCustomerDataInfrastructureNavigation'
+import { TreeMenu } from 'components/TreeMenu'
+import SEO from 'components/seo'
+import Link from 'components/Link'
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import OSTable from 'components/OSTable'
+
+const LeftSidebarContent = () => {
+    return <TreeMenu items={customerDataInfrastructureNav.children} />
+}
+
+type BIFeature = {
+    title: string
+    description: string
+    status?: 'available' | 'coming_soon'
+}
+
+const biFeatures: BIFeature[] = [
+    {
+        title: 'Table',
+        description: 'The view that real engineers prefer.',
+        status: 'available',
+    },
+    {
+        title: 'Big number',
+        description: 'Some numbers should be big.',
+        status: 'available',
+    },
+    {
+        title: 'Line chart',
+        description: 'You kow what it does.',
+        status: 'available',
+    },
+    {
+        title: 'Bar chart',
+        description: 'You can have multiple series, too.',
+        status: 'available',
+    },
+    {
+        title: 'Stacked bar chart',
+        description: 'Pretty sweet for comparing parts of a whole over time.',
+        status: 'available',
+    },
+    {
+        title: 'Area chart',
+        description: 'Fill in the area under the line.',
+        status: 'available',
+    },
+    {
+        title: 'Pivot table',
+        description: 'Summarize and aggregate data across multiple dimensions for deeper insights.',
+        status: 'coming_soon',
+    },
+]
+
+export default function BusinessIntelligence(): JSX.Element {
+    // Define table columns
+    const columns = [
+        { name: 'Feature', width: 'minmax(150px, 1fr)', align: 'left' as const },
+        { name: 'Description', width: 'minmax(300px, 2fr)', align: 'left' as const },
+        { name: '', width: '60px', align: 'center' as const },
+    ]
+
+    // Create table rows from biFeatures
+    const featureRows = biFeatures.map((feature) => ({
+        cells: [
+            { content: <span className="font-bold">{feature.title}</span> },
+            { content: feature.description, className: 'text-sm' },
+            {
+                content:
+                    feature.status === 'coming_soon' ? (
+                        <span className="rounded-sm bg-highlight py-0.5 px-1 text-xs font-bold text-red dark:text-yellow">
+                            Soon
+                        </span>
+                    ) : (
+                        '✅'
+                    ),
+                className: 'text-xl',
+            },
+        ],
+    }))
+
+    return (
+        <>
+            <SEO
+                title="Business Intelligence (BI) - PostHog data stack"
```

Comment:
I'd say this isn't released at all since we're intending BI to mean something else besides SQL -> viz?

Reply:
Jams tells me viz is 50-75% of BI... so I'm going to leave it as-is. With the updated feature set and the clarification that it's for basic use-cases I think it's fine.